### ISSUE TITLE
Minor problem disabling warnings.

### ISSUE THREAD

**Issue by jenstroeger** [yoheimuta/protolint#345]
Hello @yoheimuta, first of all, thank you for a great project and I very much appreciate your efforts!

Second, I stumbled upon a problem with disabling warnings. Take this example:
```proto
// protolint:disable MAX_LINE_LENGTH

syntax = "proto3";

/** Schnufte test enumeration. */
enum TestEnum {
  TEST_ENUM_UNSPECIFIED = 0; // Unspecified.
  FOO = 1;                               // protolint:disable:this ENUM_FIELD_NAMES_PREFIX
}
```
which results in this warning although I had disabled it at the top of the file:
```
[test.proto:8:1] The line length is 90, but it must be shorter than 80
```
If I make that an extra long _normal_ comment it works just fine; I suspect that special `protolint:` comments might collide here?

**Comment by yoheimuta** [comment]
@jenstroeger Sorry for the late response, and thank you for reaching out.

It indeed collides here. This situation isn't what I had initially anticipated, so I'm working on it.

**Comment by yoheimuta** [comment]
@jenstroeger I've fixed it and published it at https://github.com/yoheimuta/protolint/releases/tag/v0.46.3.
Can you confirm it when you have a moment?

**Comment by jenstroeger** [comment]
> Can you confirm it when you have a moment?

Great, thank you! I can confirm it’s now working 👍🏼
